[CAM-8427] - Review CAM Applications Boxes KB documentation (#2581)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Automating Deployments/managing-multi-tier-applications.md

{{{ "title": "Binding Large-Scale Deployments",
"date": "09-01-2016",
"author": "",
"date": "12-28-2018",
"author": "Julio Castanar",
"keywords": ["cam","alm","boxes", "box", "binding", "Large Deployments", "binding variables", "binding configuration", "instance connectivity"],
"attachments": [],
"contentIsHTML": false
}}}

### Table of Contents

* [Overview](#overview)
* [Audience](#audience)
* [Prerequisites](#prerequisites)
* [What are bindings?](#What-are-bindings?)
* [Which instances does a binding connect to?](#Which-instances-does-a-binding-connect-to?)
* [How to use the bindings in configuration?](#How-to-use-the-bindings-in-configuration?)
* [Bindings automatic reconfiguration](#Bindings-automatic-reconfiguration)

> * [Using bindings in three steps](#Using-bindings-in-three-steps)

Reviewer: Please, use only level 3 titles for ToC, do not add level 4 titles

    * [Define binding variables](#1.-Define-Binding-Variables)
    * [Configure bindings for your application](#2.-Configure-Bindings-for-Your-Application)
    * [Tag bindings for instance connectivity](#3.-Tag-Bindings-for-Instance-Connectivity)

### Overview

This article is meant to assist Cloud Application Manager customers who want to create and manage boxes.
It explains how to use the **Bindings**.

Bindings glue together different parts of a multi-tier application over the network. These components can be parts of an application stack, a load balancing pool, cluster, and more. Bindings enable services to auto-discover and connect dynamically at scale. They also reconfigure the instances that need it to keep their configuration up to date.

### Audience

> Cloud Application Manager Users

Reviewer: add: using ALM...

### Prerequisites

* Access to [Applications site](https://cam.ctl.io/#/boxes) (Application Lifecycle Management module) of Cloud Application Manager as an authorized user of an active Cloud Application Manager account.

### What are bindings?

> Bindings are [variables](parameterizing-boxes-with-variables.md) that you can add to boxes. They represent a connection from the deployed box to other instances. Bindings are used to program your event scripts of configuration files to work with the other instances.

Reviewer: ```suggestion
Bindings are [variables](parameterizing-boxes-with-variables.md) that you can add to boxes. They represent a connection from the deployed box to other instances. Bindings are used to customize your event scripts or configuration files to work with the other instances.
```

Reviewer: ☝️
